docs/reference/ilm/apis/slm-api.asciidoc

```diff
@@ -169,17 +197,20 @@ The output looks similar to the following:
 --------------------------------------------------
 // TESTRESPONSE[s/"modified_date": "2019-04-23T01:30:00.000Z"/"modified_date": $body.daily-snapshots.modified_date/ s/"modified_date_millis": 1556048137314/"modified_date_millis": $body.daily-snapshots.modified_date_millis/ s/"next_execution": "2019-04-24T01:30:00.000Z"/"next_execution": $body.daily-snapshots.next_execution/ s/"next_execution_millis": 1556048160000/"next_execution_millis": $body.daily-snapshots.next_execution_millis/]
 <1> The version of the snapshot policy, only the latest verison is stored and incremented when the policy is updated
-<2> The last time this policy was modified
-<3> The next time this policy will be executed
+<2> The last time this policy was modified.
+<3> The next time this policy will be executed.
+
 
-Or, to retrieve all policies:
+[[slm-api-get-all-ex]]
+===== Get all policies
 
 [source,console]
 --------------------------------------------------
 GET /_slm/policy
 --------------------------------------------------
 // TEST[continued]
 
+
```

Comment:
Do we need double spacing between things for an asciidoctor reason?

Reply:
I chiefly do this to better indicate breaks between sections. It has no impact on the final output.